src/reality-capture.ts

```diff
@@ -0,0 +1,273 @@
+}
+
+/**
+ * Enumeration for supported photoscene output formats
+ * .rcm: Autodesk Recap Photo Mesh (default)
+ * .rcs: Autodesk Recap Point Cloud^
+ * .obj: Wavefront Object
+ * .fbx: Autodesk FBX 3D asset exchange format
+ * .ortho: Ortho Photo and Elevation Map^
+ * .report: Quality Report^
+ * ^ These parameter values are available only if scenetype is set to aerial.
+ */
+export enum OutputFormat {
+    RecapPhotoMesh = 'rcm',
+    RecapPointCloud = 'rcs',
+    WavefrontObject = 'obj',
+    FBXExchangeFormat = 'fbx',
+    OrthoPhotoElevationMap = 'ortho',
+    QualityReport = 'report'
+}
+
+/**
+ * Specifies the subject type of the photoscene.
+ */
+export enum SceneType {
+    Aerial = 'aerial',
+    Object = 'object'
+}
+
+export interface IFile {
+    fileid: string;
+    filename: string;
+    filesize: string;
+    msg: string;
+}
+
+export interface IFiles {
+    photosceneid: string;
+    Files: {
+        file: IFile[];
+    }
+}
+
+export interface IPhotoScene {
+    Photoscene: {
+        photosceneid: string;
+    }
+}
+
+export interface IPhotoSceneCancelDelete {
+    msg: string;
+}
+
+export interface IPhotoSceneError {
+    code: number;
+    msg: string;
+}
+
+export interface IPhotoSceneOutput {
+    Photoscene: {
+        filesize: string;
+        photosceneid: string;
+        progress: string;
+        progressmsg: string;
+        resultmsg: string;
+        scenelink: string;
+        urn: string;
+    }
+}
+
+export interface IPhotoSceneProcess {
+    msg: string;
+    Photoscene: {
+        photosceneid: string;
+    }
+}
+
+export interface IPhotoSceneProgress {
+    photosceneid: string;
+    progressmsg: string;
+    progress: string;
+}
+
+/**
+ * Client providing access to Autodesk Forge {@link https://forge.autodesk.com/en/docs/reality-capture/v1/developers_guide/overview|reality capture APIs}.
+ * @tutorial realitycapture
+ */
+export class RealityCaptureClient extends ForgeClient {
+    /**
+     * Initializes new client with specific authentication method
+     * @param {IAuthOptions} auth Authentication object,
+     * containing `client_id` and `client_secret` properties (for 2-legged authentication).
+     * @param {string} [host="https://developer.api.autodesk.com"] Forge API host.
+     */
+    constructor(auth: IAuthOptions, host?: string) {
+        super('photo-to-3d/v1/', auth, host);
+    }
+
+    /**
+     * Creates new photoscene
+     * {@link https://forge.autodesk.com/en/docs/reality-capture/v1/reference/http/photoscene-POST|docs}.
+     * @async
+     * @param {string} scenename Specifies the name of the photoscene.
+     * @param {string} callback URI to be invoked when the processing of the photoscene has completed.
+     * @param {OutputFormat} format Output file format. Multiple file formats can be listed in a comma-delimited list.
+     * @param {SceneType} scenetype Specifies the subject type of the photoscene.
+     * @param {GpsType} gpstype Specifies the GPS coordinates type.
```

Comment:
This is not critical, but _optional_ function argument names in JsDoc are typically wrapped in square brackets, e.g., `@param {GpsType} [gpstype] Specifies the GPS coordinates type.`

Reply:
will implement the change